Personal_Accounts/src/test/java/org/openapitools/api_tests/aisp_account-consents_consentId_delete.feature

```diff
@@ -0,0 +1,41 @@
+# ********RoostGPT********
+
+# Test generated by RoostGPT for test hsbc-karate using AI Type Open AI and AI Model gpt-4
+# 
+# Feature file generated for /aisp/account-consents/{consentId}_delete for http method type DELETE 
+# RoostTestHash=ac51f8b6e8
+# 
+# 
+
+# ********RoostGPT********
+Feature: Account Consent Setup
+
+Background:
+  * def urlBase = karate.properties['url.base'] || karate.get('urlBase', 'http://localhost:8080')
+  * url urlBase
+
+Scenario Outline: Consent deletion
+  Given path 'aisp/account-consents/', <consentId>
+  And header Authorization = 'Bearer LbGDbcFVVfE-ngpjj8KaH5wrEfg'
+  And header x-fapi-auth-date = 'Sun, 10 Sep 2017 19:43:31 UTC'
+  And header x-fapi-customer-ip-address = '169.254.169.254'
+  And header x-fapi-interaction-id = <x-fapi-interaction-id>
+  And header Accept-Language = 'en-US'
+  When method delete
+  Then status <statusCode>
+  And match responseHeaders['x-fapi-interaction-id'] == <x-fapi-interaction-id>
+  # And match response == { id: '#string', errors: '#[]' } #Incorrect assertion(cannot hardcode for all requests)
+
+Examples:
+  | consentId                             | x-fapi-interaction-id                | statusCode |
+  | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 204        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 400        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 401        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 403        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 405        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 406        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 429        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 500        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 503        |
+  # | '7404e99b-1a4d-4b08-b441-c327661527f0' | 'f26da293-1fc6-4348-ac0e-f3b2f7d4ec3f' | 504        |
+  # Exactly same rows, only different status codes. Currently the mock server only supports 204 status code
```

Comment:
Since all the other columns were exactly same in all the rows, except the status codes, Commented the other rows, as mock server currently only supports 204 status code